frontend/src/components/chat/Chat.css

```
@@ -0,0 +1,78 @@
+/* .chat-container__messages-container {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  overflow-y: scroll;
+} */
+
+.chat-container {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  padding: 20px 0;
+  box-sizing: border-box;
+}
+
+.chat-container__chat-info {
+  display: flex;
+  flex-direction: column;
+  padding: 0 1rem;
+  margin-bottom: 20px;
+}
+
+.chat-container__chat-info__title {
+  font-size: 24px;
+  font-weight: bold;
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.chat-container__chat-info__event-info,
+.chat-container__chat-info__friend-info {
+  font-size: 18px;
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.chat-container__messages-container {
+  flex-grow: 1;
+  overflow-y: scroll;
+  margin-bottom: 20px;
+  padding: 0 1rem;
+}
+
+.chat-container__message-container {
+  margin-bottom: 10px;
+}
+
+.chat-container__bottom-area {
+  padding: 0 1rem;
+}
+
+.chat-container__bottom-area__disabled-message {
+  color: red;
+  margin-bottom: 10px;
+}
+
+.chat-container__bottom-area > form {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 100%;
+}
+
+.chat-container__bottom-area__message-input {
+  padding: 10px;
+  box-sizing: border-box;
+  width: 100%;
```

```suggestion
  width: 100%;
  background-color: var(--light-green);
  border-style: none;
  font-size: large;
```

The background could be set to light green and set borders to none in order to match the Figma prototype. I think font size should also be set to large.